Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -703,12 +703,15 @@ def adjustCamera(self, resetcamera= False):
 
     # Set interpolation on
     def setInterpolateOn(self):
-        self.sliceActor.SetInterpolate(True)
+        # toggle interpolation of slice actor 
+        self._viewer.imageSlice.GetProperty().SetInterpolationTypeToLinear()
         self.renWin.Render()
 
     # Set interpolation off
     def setInterpolateOff(self):
-        self.sliceActor.SetInterpolate(False)
+        # toggle interpolation of slice actor 
+        self._viewer.imageSlice.GetProperty().SetInterpolationTypeToLinear()
```

Comment:
This should be
` self._viewer.imageSlice.GetProperty().SetInterpolationTypeToNearest()`

Reply:
(to turn off the interpolation)

Reply:
Thanks, I fixed it.